src/main/java/org/apache/sling/rewriter/impl/ProcessorManagerImpl.java

```diff
@@ -84,12 +81,6 @@
     @Reference
     private ResourceResolverFactory resourceResolverFactory;
 
-    @Reference(
```

Comment:
You need to keep this, to prevent this service from being activated before the service user configuration is in place (https://sling.apache.org/documentation/the-sling-engine/service-authentication.html#serviceusermapper)

Reply:
Thanks, restored this.